### PR TITLE
Add John Hancock (jhpensions.com) to unsupported services list

### PR DESCRIPTION
**Pull request by rwinikates**

### CODE REVIEW

File: _data/investing.yml [hunk 1]
```diff
@@ -57,6 +57,12 @@ websites:
       software: Yes
       doc: https://www.interactivebrokers.com/en/index.php?f=ibgStrength&p=log
 
+    - name: John Hancock
+      url: https://www.jhpensions.com
```
Comment (Jawshy) [inline _data/investing.yml]: I cannot access this domain and neither can [Is It Down Right Now](http://www.isitdownrightnow.com/jhpensions.com.html).

**Review by Jawshy — CHANGES_REQUESTED** [review]

File: _data/investing.yml (hunk 1 above)
Comment (Jawshy) [reply _data/investing.yml]: Did you mean to write `https://www.jhancockpensions.com/`, @rwinikates?

Comment (rwinikates) [reply _data/investing.yml]: @Jawshy yep! jhpensions.com 301s me to that full url, apologies for the issues. Will fix.

**Review by stephengroat — APPROVED** [review]

**Review by Jawshy — CHANGES_REQUESTED** [review]
The Twitter handle and Facebook page both link to `http://www.johnhancock.com/` not `https://www.jhancockpensions.com`. :confused:

**Comment by Carlgo11** [comment]
Closing due to inactivity.

**Pull request closed without merging**